system-parachains/bridge-hubs/bridge-hub-kusama/src/lib.rs

```diff
@@ -472,6 +507,179 @@ impl pallet_utility::Config for Runtime {
 	type WeightInfo = weights::pallet_utility::WeightInfo<Runtime>;
 }
 
+// Ethereum Bridge
+parameter_types! {
+	pub storage EthereumGatewayAddress: H160 = H160(hex_literal::hex!("EDa338E4dC46038493b885327842fD3E301CaB39"));
+}
+
+parameter_types! {
+	pub const CreateAssetCall: [u8;2] = [53, 0];
+	pub const CreateAssetDeposit: u128 = (UNITS / 10) + EXISTENTIAL_DEPOSIT;
+	pub const InboundQueuePalletInstance: u8 = system_parachains_constants::kusama::snowbridge::INBOUND_QUEUE_PALLET_INDEX;
+	pub Parameters: PricingParameters<u128> = PricingParameters {
+		exchange_rate: FixedU128::from_rational(1, 400),
+		fee_per_gas: gwei(20),
+		rewards: Rewards { local: 1 * UNITS, remote: meth(1) }
```

Comment:
Is the local reward  too much here?

Reply:
Good catch @yrong! Updated here: https://github.com/polkadot-fellows/runtimes/pull/130/commits/d44fa3305087752131d651adf120a1390a222667

Reply:
IIUC the local reward for single message is still `1 KSM + <length-dependent>`. Is it intended? I mean - `1 KSM` for every message seems too large to me. Or am I missing something?

Reply:
Also: IIUC this value is correlated with the `BridgeHubEthereumBaseFee` at sibling asset hub. I mean - we expect sender to pay at least this reward to AH first and then sovereign account of AH to pay it to the relayer, right?

If so, can we please document it somewhere in the code? Just because someday someone may want to update pricing parameters here, at BH and forget to update base fee at the AH => AH will start paying to relayer out of his pocket. At least some code warning aside both values may add some protection against that in the future.

Reply:
The `PricingParameters::rewards::local` config field is used to calculate the combined refund+reward to pay relayers for delivering messages to BridgeHub from Ethereum. So its not associated with `BridgeHubEthereumBaseFee`.

On the other hand the `PricingParameters::rewards::remote` config field is used to calculate the combined refund+reward to pay relayers for delivering messages to Ethereum from BridgeHub, so it is associated with `BridgeHubEthereumBaseFee`.

Since the bridge is disabled on initial launch, none of these hardcoded pricing parameters need to be accurate. They will updated post launch via a governance proposal.

However, I suggest changing `PricingParameters::rewards::local` to `0.01 KSM`, which is probably what we will use anyway for the initial bridge activation proposal.

@svyatonik I'll add more comments in the runtime config and `PricingParameters` struct describing these fields and their relationships.

Reply:
@svyatonik @vgeddes I decreased the local fee to `0.01` KSM: https://github.com/polkadot-fellows/runtimes/pull/130/commits/042aa1b6f3ea74bf7118b42ed86254f9acdd7f47#diff-da00a6055ce091ceb6769ed5cc0dc6527aca586de0394a60aa3d18c38ff89716R523